xatlas.cpp:166:32: error: use of undeclared identifier 'realloc'
(happened on llvm-musl with libcxx-23, but haven't narrowed down the cause)
--- a/src/3rdparty/xatlas/xatlas.cpp
+++ b/src/3rdparty/xatlas/xatlas.cpp
@@ -52,2 +52,3 @@
 #include <stdint.h>
+#include <stdlib.h>
 #include <stdio.h>
